package.json

```diff
@@ -23,7 +23,7 @@
     "mkdirp": "^0.5.1",
     "pify": "^2.3.0",
     "rimraf": "^2.5.4",
-    "yargs": "^5.0.0"
+    "yargs": "5.0.0"
```

Comment:
Because I'm moving fast and breaking things? I didn't see the dependency was already in there, and I npm installed it. Probably overwrote it.